cluster/examples/vmi-preset-virtiorng.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kubevirt.io/v1alpha2
```

Comment:
I thought we had the plan to add it by default, why do we need preset for it?

Reply:
This is just example present if you want to create one. I find it handy, but can be removed.